demos/question-answering/02.serve-vectorstore.ipynb

```diff
@@ -20,7 +20,7 @@
     "\n",
     "In this Notebook, you will be taking the next step in this journey. You will use MLFlow to log the Chroma DB files as artifacts of an experiment. After logging the artifacts, you will then create an Inference Service that retrieves these artifacts and uses them to provide context to user queries. For this purpose, you'll be using KServe, a Kubernetes-based platform that provides a serverless framework for serving machine learning models at scale.\n",
     "\n",
-    "It's important to note that since KServe does not support serving Chroma DB files out-of-the-box, you will be using a custom predictor component. This means that you'll need to create a Docker image first, which can then be deployed as our Inference Service. This process allows for a high degree of customization, enabling you to fine-tune your service to your specific needs. You can find the code, as well as the Dockerfile for the custom predictor inside the `vectorstore` directory of this project. However, to save time, you can use the one we have pre-built for you: `gcr.io/mapr-252711/ezua-demos/vectorstore:v0.1.0`.\n",
+    "It's important to note that since KServe does not support serving Chroma DB files out-of-the-box, you will be using a custom predictor component. This means that you'll need to create a Docker image first, which can then be deployed as our Inference Service. This process allows for a high degree of customization, enabling you to fine-tune your service to your specific needs. You can find the code, as well as the Dockerfile for the custom predictor inside the `dockerfiles/vectorstore` directory of this project. However, to save time, you can use the one we have pre-built for you: `dpoulopoulos/qna-vectorstore:c263bf6`.\n",
```

Comment:
Do you think we can put the image name, image tag and other constants in .env or .toml? We can hide this file with a dot like .constants.env

Reply:
This is temporary. We want to get our hands on a public image registry that we will use for the tutorials. So, we will use our own `gcr.io` registry soon. However, we cannot use any of the existing ones yet, since they are not public and this causes several issues with customer clusters.